integration-test/src/main/java/org/apache/iotdb/it/env/cluster/env/AbstractEnv.java

```diff
@@ -1007,4 +1009,22 @@ public String getToolsPath() {
   public String getLibPath() {
     return TEMPLATE_NODE_LIB_PATH;
   }
+
+  @Override
+  public Optional<DataNodeWrapper> dataNodeIdToWrapper(int nodeId) {
+    try (SyncConfigNodeIServiceClient leaderClient =
+        (SyncConfigNodeIServiceClient) getLeaderConfigNodeConnection()) {
+      TShowDataNodesResp resp = leaderClient.showDataNodes();
```

Comment:
Why get DataNode through cluster rpc? The DataNodeWrapper should exist in AbstractEnv already

Reply:
Because DataNodeWrapper not contains node id